src/CollisionLayer.js

```diff
@@ -17,13 +18,30 @@ module.exports = class CollisionLayer {
     return this.collisionsMap[posx + posy * this.width]
   }
 
-  constructCollisionsMap(tilesMap) {
-    this.collisionsMap = new Array(tilesMap.length)
+  constructCollisionsMap() {
+    this.collisionsMap = new Array(this.tilesMap.length)
 
-    for (let i = 0; i < tilesMap.length; ++i) {
-      let tile = tilesMap[i]
+    for (let i = 0; i < this.tilesMap.length; ++i) {
+      let tile = this.tilesMap[i]
 
       this.collisionsMap[i] = (tile === undefined)
     }
   }
-}
\ No newline at end of file
+
+  getCollidables(){
+    let r=0, c=0,
+        collidables = []
+    for (let i = 0; i < this.tilesMap.length; ++i) {
+      c++;
+      if(i%50==1){
```

Comment:
If I'm not wrong you are checking if you are in a new row, but rows are not always of size 50. Should not this element be changed by `this.width` to check the map width?